src/Octoshift/Services/OctoLogger.cs

```diff
@@ -156,5 +155,17 @@ public virtual void LogSuccess(string msg)
         Console.ResetColor();
     }
 
-    public virtual void RegisterSecret(string secret) => _secrets.Add(secret);
```

Comment:
We could also avoid this refactor, and keep a separate list of `_secrets` which we replace as well as the "redaction patterns". That's probably a safer approach to avoid surprises - but I'd love to hear what y'all think.